gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/UserQuotaManager.java

```diff
@@ -64,7 +64,7 @@
    * Checks if the dagNode exceeds the statically configured user quota for both the proxy user and requester user
    * @throws IOException if the quota is exceeded, and logs a statement
    */
-  public void checkQuota(Dag.DagNode<JobExecutionPlan> dagNode) throws IOException {
+  public void checkQuota(Dag.DagNode<JobExecutionPlan> dagNode, boolean onStartup) throws IOException {
```

Comment:
strictly this is on leader change / transfer to active, not startup per se, correct?  maybe `withinInit`?